src/index.tsx

```diff
@@ -403,12 +424,17 @@ class App extends React.Component<{}, AppState> {
       });
       drawScene();
       event.preventDefault();
-    } else if (event.key === "a" && event.metaKey) {
+    } else if (event.metaKey && event.key === "a") {
```

Comment:
`event.metaKey` is suitable even for windows and linux users?

Reply:
No idea...